docs/chapters/ci-and-cd/pelux-ci-cd-jobs.rst

```diff
@@ -0,0 +1,96 @@
+PELUX Continuous Integration jobs
+=================================
+
+For the PELUX Baseline, the Jenkins instance at `pelux.io/jenkins
+<http://pelux.io/jenkins>`_ has some predefined jobs that we run regularly or on
+commit.
+
+General strategy
```

Comment:
I see some potential for Blueprint content here. Could we have the entrypoint to CI&CD in the Blueprint and put some of these more general considerations there and then link to the Baseline SWF docs? That entrypoint would be good to have for other users and would serve as a place to put further content on the subject.

Reply:
I agree, some of the top matter in the article should go into the blueprint. I do think we agreed not to link in that direction (blueprint -> deployment), but it doesn't really matter, one can easily link the other way around by pulling in the article.